### ISSUE TITLE
Running Code not showing compile errors in subclasses

### ISSUE THREAD

**Issue by mrbdahlem** [PerryHighCS/HelloFunction#13]
HelloFunction is not failing on compile step. Instead it is continuing and attempts to use classes that did not compile.

[This sample](https://run.mycode.run/?code=XQAAAQANBgAAAAAAAAA9iIpHPJkiDSD3eirckGfVDlyCzGKafDiNdzYcptq24WYsABZXK1m1qc4UVyDjDdEKGWHLaCKmwJfhoHyYuO72KsNT9qshyUreD2TO%2FvLtLZTecsJMzRe2Akzc%2FYNAZrHv5EunBd%2BbziRFH6VAimVICK3cS49YWeXBFb1h%2FWFQTIk0QkJeiXEoaTNovElxIJdAiDLNrjDxVTgbgl9hRS6lfK5tVYzkxR5eCYM%2BSZ%2BdnrBWlYBPMB%2FGFJQExc2kYrE26ccSfCR03LtYmZzLHRL5FIKm9sSaz8DWvUU2EBqjDQzNLwGFcAcFrbob2cn70WR13Wwt%2F3zVhrfit71DKRxeDFJb1Xn%2B5ROcjnah4zQcp8TZ5v%2FRacYQE4LA7lsteFzW5E6DbfINbIC%2F6KGLjdMwQvfh%2B4BZerVVVOvg3o9h77YO4V5OriXnqJyePxXEsf3wwFSGxO6VfSZUt8loIMUHYmCfvu3eI%2Bb7eYCUVczQ8fQZYYE2KsciQeFeSErgqU6XBclrRB8orF%2BaXtAyGkUoFVJ%2BtFF3YacIGkZdpUW52G9Gb80Uj3cLsXjMvApgjtKYyuaxpbG3YYoXCgJ54%2ForO%2FTjbVFI7G0H%2BE2hVlSZs8Zp3Z1pX0NSB4kyehQS9sOk4R%2ByoTNR5Xb430wA4M0XrUUj%2BQIyXQ%3D%3D&noproject=true) should give an error ```Implicit super constructor Rectangle() is undefined. Must explicitly invoke another constructor```, but does not return it. Instead in the Lambda logs we see:

```Square: java.lang.NoClassDefFoundError
java.lang.NoClassDefFoundError: Square
	at java.lang.Class.getDeclaredMethods0(Native Method)
	at java.lang.Class.privateGetDeclaredMethods(Class.java:2701)
	at java.lang.Class.privateGetMethodRecursive(Class.java:3048)
	at java.lang.Class.getMethod0(Class.java:3018)
	at java.lang.Class.getMethod(Class.java:1784)
	at run.myCode.compiler.CodeRunner.runIt(CodeRunner.java:34)
	at run.myCode.Hello.handleRequest(Hello.java:123)
Caused by: java.lang.ClassNotFoundException: Square
	at java.net.URLClassLoader.findClass(URLClassLoader.java:382)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	at run.myCode.compiler.FromMemoryClassLoader.findClass(FromMemoryClassLoader.java:34)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:424)
	at java.lang.ClassLoader.loadClass(ClassLoader.java:357)
	... 7 more
```

**Comment by mrbdahlem** [comment]
Attempting to solve with an additional catch in runIt.

**Comment by mrbdahlem** [comment]
Appears solved in commit 2f3b1f5
